vertical-pod-autoscaler/pkg/recommender/main.go

```diff
@@ -62,9 +62,12 @@ var (
 	prometheusAddress      = flag.String("prometheus-address", "http://prometheus.monitoring.svc", `Where to reach for Prometheus metrics`)
 	prometheusJobName      = flag.String("prometheus-cadvisor-job-name", "kubernetes-cadvisor", `Name of the prometheus job name which scrapes the cAdvisor metrics`)
```

Comment:
Would it make sense to move these two flags into the prometheus block also? May be put them at the start.